score/README.md

```diff
@@ -18,6 +18,16 @@
 - Activate your virtualenv (see above)
 - Type `jupyter notebook`. Your browser should open with a Jupyter Notebook tab
 
+## Activating variable-enabled Markdown for Jupyter notebooks
+- Change to this directory (i.e. `cd score`)
+- Run `jupyter contrib nbextension install --user`
+- Run `jupyter nbextension enable python-markdown/main`
+- Make sure you've loaded the Jupyter notebook in a "Trusted" state. (See button near
+ top right of Notebook screen.)
+
+For more information, see [nbextensions docs](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html) and
+see [python-markdown docs](https://github.com/ipython-contrib/jupyter_contrib_nbextensions/tree/master/src/jupyter_contrib_nbextensions/nbextensions/python-markdown).
+
```

Comment:
I would move this section to the score comparison notebook itself, since this pertains specifically to generate the markdown cell, which only pertains to people who runs this, which I believe are going to be a small amount of people. @lucasmbrown-usds thoughts?

Reply:
Hmm. I mean, we also have the jupyter notebook setup instructions in that same place, so it feels like grouping like with like, and I generally prefer to have setup instructions in just one place. 

I could add a sentence that says "currently only used by the Notebook `scoring_comparison.ipynb`?

Reply:
Yeah, that works.